src/views/HomeView.vue

```diff
@@ -19,25 +21,33 @@ const correctAnswer = ref("");
 const answeredQuestion = ref(false);
 const answeredCorrectly = ref(false);
 
-
-function checkAnswer(answer){
+const checkAnswer = (answer) => {
   answeredQuestion.value = true;
 
-  if(answer === correctAnswer.value){
+  if (answer === correctAnswer.value) {
     answeredCorrectly.value = true;
-  }
-  else{
+  } else {
```

Comment:
Are you using automatic formatting of code at saving? If not let me know and I will help set this up, helps keeping the code uniform